python/ray/scripts/scripts.py

```diff
@@ -654,11 +656,14 @@ def start(node_ip_address, address, port, redis_password, redis_shard_ports,
             with cli_logger.indented():
                 cli_logger.print("{} ray", cf.magenta("import"))
                 cli_logger.print(
-                    "ray{}init(address{}{}{})", cf.magenta("."),
+                    "ray{}init(address{}{}{}{})", cf.magenta("."),
```

Comment:
In the case of joining an existing cluster (`address="auto"`), it might be possible to make the `_node_ip_address` parameter unnecessary, by reading the value from a running Raylet with `_find_address_from_flag()`. Ok to just leave a note or todo.

Reply:
Good suggestion. Add a note here https://github.com/ray-project/ray/pull/20720/files#diff-15b2fb983deb109a406d72dff4c44e7d063a8274c6bfedaea2f29697066511ccR659.